roles/network_plugin/calico/tasks/check.yml

```diff
@@ -173,3 +173,14 @@
     - typha_enabled
   run_once: True
   delegate_to: "{{ groups['kube_control_plane'][0] }}"
+
+- name: "Check ipip and vxlan mode both Nerver for calco ipv6"
```

Comment:
https://github.com/projectcalico/calico/blob/330e51bea4c450137992f13cb7d2c92ebdeffae2/libcalico-go/lib/clientv3/ippool.go#L433 shows `IPIP cannot be enabled for IPv6.`
Why here checks VXLAN also?

Reply:
Yes, the latest version supports it.

https://github.com/projectcalico/calico/pull/5961/files